app/assets/javascripts/modules/accordion-with-descriptions.js

```diff
@@ -212,6 +212,7 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       var shouldUpdateHash = true;
 
       this.title = $subsectionElement.find('.js-subsection-title').text();
+      this.element = $subsectionElement;
```

Comment:
`this` is really confusing in JavaScript, so I prefer to avoid where possible. Also, `this.element` wouldn't work in the `function`s in this object, so you'd _still_ need the `$subsectionElement` variable in places and I'd rather use that than the confusing `this.element` call.